src/components/Tool/Appeal/Flow/ContactFlowColumn/ContactFlowColumn.test.tsx

```diff
@@ -95,4 +101,24 @@ describe('ContactFlowColumn', () => {
       getByRole('menuitem', { name: 'Select 1 contact' });
     });
   });
+
+  it('should have "Add Contact to Appeal" button', async () => {
```

Comment:
```suggestion
  it('asked column should have "Add Contact to Appeal" button', async () => {
```

Reply:
I've change it to the following so it flows with the `it` at the beginning.

```it('should ensure the "asked" column has the "Add Contact to Appeal" button', async () => {```

Reply:
Actually, I don't like how it reads, as it makes it more confusing. I'll do what you suggested.